### PR TITLE
Update files as per requirement

### PR DESCRIPTION
**Pull request by mahbubul14**
## Project requirements

* Create a class called Solver.
* Create a method called factorial that takes one argument, an integer N, and returns the factorial for that number. The factorial is the multiplication of all integers from 1 to N and has the special case that the factorial of 0 is 1. This method only accepts 0 and positive integers, so if a negative integer is given it should raise an exception.
* Create a method called reverse that takes one argument, a string word, and returns word reversed (e.g. if word is "hello" it returns "olleh").
* Create a method called fizzbuzz that takes one argument, an integer N, and returns a string. The returned string is constructed following these rules:
  * When N is divisible by 3, return "fizz".
  * When N is divisible by 5, return "buzz".
  * When N is divisible by 3 and 5, return "fizzbuzz".
Any other case, return N as a string (e.g. say N is 7 then return "7").
* Remember to start with tests for all of the methods described above. Your commit history should prove that you have used TDD.

### CODE REVIEW

File: spec/solver_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  describe '#factorial' do
+    it('should check of factorial of a number') do
+      expect(@solver.factorial(3)).to eq 6
+    end
+
+    it('should check of factorial of a number') do
+      expect(@solver.factorial(5)).to eq 120
+    end
+  end
```
Comment (Dipeshtwis) [inline spec/solver_spec.rb]: - [x] I would recommend testing the Special case - when input is 0.

File: spec/solver_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      expect(@solver.fizzbuzz(15)).to eq 'fizzbuzz'
+    end
+
+    it 'Should verify if the number is multiple of 3' do
+      expect(@solver.fizzbuzz(6)).to eq 'fizz'
+    end
+
+    it 'Should verify if the number is multiple of 5' do
+      expect(@solver.fizzbuzz(10)).to eq 'buzz'
+    end
```
Comment (Dipeshtwis) [inline spec/solver_spec.rb:33]: - [x] I would recommend testing the case where the number does not belong to `fizz`, `buzz`, or `fizzbuzz`.

```
expect(@solver.fizzbuzz(7)).to eql('7')
```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mahbubul14 and @Rayan84, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [x] It's a small change but it is very important for you to provide all the commands about your project for the other users. In the README file, Please mention the step on how to test the project. [HInt: `rspec`]

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @mahbubul14 and @Rayan84  

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**